Source/WebKit/UIProcess/WebAuthentication/Cocoa/LocalAuthenticator.mm

```diff
@@ -86,10 +86,20 @@ static bool shouldUpdateQuery()
 using namespace WebAuthn;
 using CBOR = cbor::CBORValue;
 
+BOOL shouldUseAlternateKeychainAttribute()
+{
+#if USE(APPLE_INTERNAL_SDK)
```

Comment:
`USE(APPLE_INTERNAL_SDK)` in an implementation file is an anti-pattern.

Since you already have the `respondsToSelector` check, why is it necessary? The method could be forward declared for Open Source builds.

Reply:
Aditya has a good point. We should probably be gating this with a `HAVE()` macro. uses of `getASCWebKitSPISupportClass()` happen outside of `USE(APPLE_INTERNAL_SDK)` in other WebAuthn code, so this seems wrong.

Reply:
And add `shouldUseAlternateKeychainAttribute` to the forward declarations in `WebKit/platform/spi/Cococa/AuthenticationServiesCoreSPI.h`

Reply:
The forward declaration of `ASCWebKitSPISupport` is gated behind an `||` of three feature flags. I've gone with `HAVE(UNIFIED_ASC_AUTH_UI)` for my code here to replace `USE(APPLE_INTERNAL_SDK)`